ant/base/ant.py

```diff
@@ -231,8 +232,13 @@ def read_message(self):
     def unassign_channel(self, channel):
         pass
 
-    def assign_channel(self, channel, channelType, networkNumber):
-        message = Message(Message.ID.ASSIGN_CHANNEL, [channel, channelType, networkNumber])
+    def assign_channel(self, channel, channelType, networkNumber, ext_assign):
+        print("ext_assign")
+        print(ext_assign)
```

Comment:
Remove prints

Reply:
sorry, forgot to remove the lines for testing...